templates/layout.html

```diff
@@ -63,7 +63,10 @@ <h2>What's this thing?</h2>
 			<p>This reader was born while developing Guya.moe, the website to host Kaguya. After a while we added a lot of useful features, and now we branched it into a separate domain. We don't host anything, this is just a web image viewer that can parse other websites.</p>
 
 			<h2>What are the sites you can proxy?</h2>
-			<p>Currently it supports imgur, nhentai and custom appropriately-formatted JSON hosted on GitHub Gists. This is an advanced feature and will be explained in detail below.</p>
+			<p>Currently it supports imgur, OneDrive, nhentai and custom appropriately-formatted JSON hosted on GitHub Gists. This is an advanced feature and will be explained in detail below.</p>
+			
+			<h2>Disclaimer regarding OneDrive shares</h2>
+			<p>OneDrive links provide user information (look for the "More Details" pane on the "Info" tab).<br>Keep that in mind if you intend to share Cubari links from that source.<br>We can't opt-out of receiving that information, so the best we can do is pinky-swear we don't even read it.</p>
```

Comment:
I have a little fud about burying this in our "about" page wall of text when we're not the only ones that can read this information. The share URL should have a "source" button that reveals the original resource:

<img width="407" alt="image" src="https://github.com/subject-f/cubarimoe/assets/52957110/d54c5856-2e07-4916-95aa-07815e1d6440">

That original URL should in some way reveal the original source, which would also reveal the drive's owner.

I've seen gist owners not realizing that their GitHub accounts are publicly visible so I'm still a bit wary about this.

Reply:
I tried hijacking the function to add a new pop-up element exclusively for this disclaimer, but my frontend-fu wasn't strong enough. IIRC I managed to make it show up but broke the close button 😅.